ExpandableText: Adds the expandable text component previously known as truncate component. (#39)
patternfly/react-component-groups: packages/module/patternfly-docs/content/extensions/component-groups/examples/ExpandableText/ExpandableText.md

---
section: extensions
subsection: Component groups

> id: ExpandableText

Reviewer: ```suggestion
id: Expandable text
```

Author: Done.

source: react
propComponents: ['ExpandableText']
---

import ExpandableText from "@patternfly/react-component-groups/dist/dynamic/ExpandableText";

## Component usage

> ### ExpandableText component

Reviewer: We should probably add some description similar tho the rest of the components

Author: Done.

---
section: extensions
subsection: Component groups

> id: Expandable Text

Author: done

source: react
propComponents: ['ExpandableText', 'ExpandableTextCustomButtonProps']
---

import ExpandableText from "@patternfly/react-component-groups/dist/dynamic/ExpandableText";

> ## Component usage

Reviewer 2: ```suggestion
## Examples
```

The **expandable text** component enables you to truncate text and display the full text via a button. The button can be displayed inline with the text or below the text. The button can be customized to display custom text or a custom button.  Alternatively you can display the full text on hover.

> 

Reviewer 2: ```suggestion

## Examples

```

Reviewer 2: moving below the "intro" text

### Basic ExpandableText component

This is an example of a basic ExpandableText component. It will truncate the text and display the full text when the button is clicked.  The button will be display under the text and will toggle the text.

```js file="./ExpandableTextExample.tsx"

```

### ExpandableText component with inline text

This is an example of a ExpandableText component with inline text. It will truncate the text and display the full text when the button is clicked. The button will be display inline with the text and will toggle the text.

```js file="./ExpandableTextInlineExample.tsx"

```

> ### ExpandableText component with custom text
> 
> This is an example of a ExpandableText component with custom text. It will truncate the text and display the full text when the button is clicked.

Reviewer 2: ```suggestion
### With custom button text

You can customize the button text to fit your needs with the `customButton` property. 

The following example customizes the text to change when expanded and collapsed.

```
